version: '3.9'

services:
  web:
    build:
      context: .
    command: python manage.py runserver 0.0.0.0:8000
    volumes:
      - .:/app
    ports:
      - "8000:8000"
    depends_on:
      - db

  db:
    image: postgres:15
    environment:
      POSTGRES_DB: site_db
      POSTGRES_USER: site_user
      POSTGRES_PASSWORD: site_pass  # ← тут была ошибка
    volumes:
      - postgres_data:/var/lib/postgresql/data/

volumes:
  postgres_data:
